shared/shot.js

```diff
@@ -334,9 +334,9 @@ class AbstractShot {
   }
 
   get isOwner() {
-    if (this._isOwner === undefined) {
+    /*  if (this._isOwner === undefined) {
```

Comment:
@ianb With this PR we are allowing isOwner to be undefined thast why commenting explicit throw of error here.

Reply:
Could you explain why this change is needed? Is there a particular error this works around?

Reply:
This was failing in extension while taking shot